bindings/c/src/lib.rs

```diff
@@ -46,6 +46,26 @@ pub unsafe extern "C" fn libsql_sync(
     }
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn libsql_sync2(
+    db: libsql_database_t,
+    out_replicated: *mut replicated,
+    out_err_msg: *mut *const std::ffi::c_char,
+) -> std::ffi::c_int {
+    let db = db.get_ref();
+    match RT.block_on(db.sync()) {
+        Ok(replicated) => {
+            (*out_replicated).frame_no = replicated.frame_no().unwrap_or(0) as i32;
```

Comment:
Shouldn't we check if out_replicated is not null?

Reply:
Added that check thanks